src/test/test_api.py

```diff
@@ -38,68 +45,94 @@ def save_test_docs(count, edges=False):
+            headers=headers_admin,
+            params={'reset': '1', 'init_collections': '1'}
+        )
+
     def test_root(self):
         """Test root path for api."""
-        resp = requests.get(url).json()
+        resp = requests.get(url + '/').json()
         self.assertEqual(resp['arangodb_status'], 'connected_authorized')
         self.assertTrue(resp['commit_hash'])
         self.assertTrue(resp['repo_url'])
 
     def test_update_specs(self):
+        """Test the endpoint that triggers an update on the specs."""
         resp = requests.get(
             url + '/api/update_specs',
-            headers={'Authorization': 'Bearer ' + auth_token},
+            headers=headers_admin,
             params={'reset': '1', 'init_collections': '1'}
         )
         resp_json = resp.json()
         self.assertEqual(resp.status_code, 200)
         self.assertTrue(len(resp_json['status']))
 
     def test_list_views(self):
+        """Test the listing out of saved AQL views."""
         resp = requests.get(url + '/api/views').json()
         self.assertTrue('list_all_documents_in_collection' in resp)
 
     def test_show_view(self):
+        """Test the endpoint that displays AQL source code for one view."""
         resp = requests.get(url + '/api/views/count_documents_in_collection').text
         self.assertTrue('Return count of documents' in resp)
 
     def test_list_schemas(self):
+        """Test the listing out of registered JSON schemas for vertices and edges."""
         resp = requests.get(url + '/api/schemas').json()
         self.assertTrue('example_vertices' in resp['vertices'])
         self.assertTrue('example_edges' in resp['edges'])
         self.assertFalse('error' in resp)
         self.assertTrue(len(resp))
 
     def test_show_schema(self):
+        """Test the endpoint that displays the JSON source for one schema."""
         resp = requests.get(url + '/api/schemas/example_edges').text
         self.assertTrue('_from' in resp)
         resp = requests.get(url + '/api/schemas/example_vertices').text
         self.assertTrue('_key' in resp)
 
-    def test_save_documents_no_auth(self):
-        # Missing bearer
-        resp = requests.put(url + '/api/documents?on_duplicate=error&overwrite=true&collection').json()
-        self.assertTrue('Missing header' in resp['error'])
-        # Invalid bearer
+    def test_save_documents_missing_auth(self):
+        """Test an invalid attempt to save a doc with a missing auth token."""
+        resp = requests.put(
+            url + '/api/documents?on_duplicate=error&overwrite=true&collection'
+        ).json()
+        self.assertEqual(resp['error'], 'Missing header: Authorization')
+
+    def test_save_documents_invalid_auth(self):
+        """Test an invalid attempt to save a doc with a bad auth token."""
         resp = requests.put(
             url + '/api/documents?on_duplicate=error&overwrite=true&collection',
             headers={'Authorization': 'Bearer xyz'}
```

Comment:
Pedantic, but if you define an 'invalid_token' variable, I'd expect it to be used here